extensions/typescript-language-features/src/utils/fileSchemes.ts

```diff
@@ -13,12 +13,14 @@ export const vscodeNotebookCell = 'vscode-notebook-cell';
 export const memFs = 'memfs';
 export const vscodeVfs = 'vscode-vfs';
 export const officeScript = 'office-script';
+export const vscodeWeb = 'vscode-test-web'; // TODO: Should probably reuse vscodeVfs eventually, since vscode-test-web is the local test web server
```

Comment:
Or web, can we make `semanticSupportedSchemes` include the scheme of all workspace folders instead of hardcoding this?